fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java

```diff
@@ -342,16 +376,15 @@ public boolean visit(String elementName, int elementIndex, Uri uri) {
                 throw invalidComboException(searchParamType, uri);
             }
 
-            // For REFERENCE search parameters, we need to treat Uris as tokens,
+            // For REFERENCE search parameters, we need to treat Uris as references,
```

Comment:
Do you happen to have any example of elements of type `uri` that would get indexed by search parameters of type `reference`?  I know the spec says its allowed, but I was having trouble wrapping my head around it and I think a concrete example might help me.